front/app/components/PageForm/index.tsx

```diff
@@ -113,7 +114,12 @@ const PageForm = ({
 
   const renderFileUploader = (props: FieldProps) => {
     return (
-      <FormikFileUploader resourceId={pageId} resourceType="page" {...props} />
+      <FormikFileUploader
+        id={uuidv4()}
```

Comment:
Need the uuid because when we have a new page, there's no data yet (like pageId, pageSlug) I can use to form a unique idea

Reply:
The id is needed to allow multiple page forms to be opened at /admin/settings/pages (otherwise the label gets confused if multiple inputs have the same id.